ykrt/src/compile/jitc_llvm/mod.rs

```diff
@@ -284,7 +282,7 @@ impl Compiler for JITCLLVM {
             // The LLVM backend is now legacy code and is pending deletion, so it's not worth us
             // spending time auditing all of the failure modes and categorising them into
             // recoverable/temporary. So for now we say any error is temporary.
-            Err(CompilationError::Temporary("llvm backend error".into()))
+            Err(CompilationError("llvm backend error".into()))
```

Comment:
I can't see which variant of `CompilationError` is being constructed any more. Which is it making?

Reply:
https://github.com/ykjit/yk/pull/1040/commits/72162f01dcb735bdbaebdc860b8b49dc1c4da8d1#diff-fa66cad0aec598e0ed07d8166314f32521aca9b43e97016babccc058de8ddd00R26

Reply:
Ah. Github led me astray. If you click the type it shows the old definition.